### PR TITLE
address cran comments

### PR DESCRIPTION
**Pull request by LittleBeannie**

### CODE REVIEW

File: R/prepare_ae_specific_subgroup.R [hunk 1]
```diff
@@ -30,14 +30,17 @@
...
 #' @examples
-#' # meta <- meta_ae_example()
-#' # prepare_ae_specific_subgroup(
-#' #   meta,
-#' #   population = "apat",
-#' #   observation = "wk12",
-#' #   parameter = "rel",
-#' #   subgroup_var = "RACE"
-#' # )$data
+#' \dontrun{
```
Comment (LittleBeannie) [inline R/prepare_ae_specific_subgroup.R:33]: This example gives an error message `"Error: This function is still under development."`, so I saved it in a `\dontrun{}` environment.

**Review by nanxstats — CHANGES_REQUESTED** [review]
I think these changes need further review.

**Review by elong0527 — APPROVED** [review]
Added suggestions to address CRAN comments. 

@nanxstats , could you review the changes before we merge? Thanks!

File: DESCRIPTION [hunk 2]
```diff
@@ -18,8 +18,7 @@ Description: Analyzes adverse events in clinical trials using the 'metalite'
     data structure. The package simplifies the workflow to create
     production-ready tables, listings, and figures discussed in the
     adverse events analysis chapters of
-    "R for Clinical Study Reports and Submission"
-    (Zhang et al., 2022, <https://r4csr.org/>).
+    Zhang et al., 2022, <https://r4csr.org/>.
```
[the remark this answers is not in the document]
Comment (nanxstats) [reply DESCRIPTION]: The reviewer meant the reference format should be changed from `author, 2022, <url>` to `author, (2022) <url>`, not about the reference itself.

File: R/prepare_ae_specific_subgroup.R (hunk 1 above)
Comment (nanxstats) [reply R/prepare_ae_specific_subgroup.R]: Avoid using `\dontrun` as they actually will be ran and they are likely be asked to be removed.

I see Yilong added a patch to replace `stop()` with `message()` and return `NULL` early which is the right way to go.

File: R/n_subject.R [hunk 3]
```diff
@@ -77,12 +77,12 @@ n_subject <- function(id, group, par = NULL, use_na = c("ifany", "no", "always")
...
-#' # library(r2rtf)
-#' #
-#' # r2rtf_adae$TRTA <- factor(r2rtf_adae$TRTA)
-#' #
-#' # metalite.ae:::avg_event(r2rtf_adae$USUBJID, r2rtf_adae$TRTA)
-#' # metalite.ae:::avg_event(r2rtf_adae$USUBJID, r2rtf_adae$TRTA, r2rtf_adae$AEDECOD)
+#' \donttest{
+#' library(r2rtf)
+#' r2rtf_adae$TRTA <- factor(r2rtf_adae$TRTA)
+#' metalite.ae:::avg_event(r2rtf_adae$USUBJID, r2rtf_adae$TRTA)
```
[the remark this answers is not in the document]
Comment (nanxstats) [reply R/n_subject.R:83]: I removed the `\donttest` here as the `@noRd` here will not generate an `Rd` file.

**Review by nanxstats — APPROVED** [review]
I added a few minor modifications and now it looks ready. Thanks for patching these up!

**Pull request merged**